common/static/common/js/components/views/feedback.js

```diff
@@ -65,6 +67,39 @@
                     return this;
                 },
 
+                inFocus: function() {
+                    this.options.outFocusElm = this.options.outFocusElm || document.activeElement;
+
+                    // Set focus to the container.
+                    $(this.el).find(".wrapper").first().focus();
+
+                    /**
```

Comment:
Nit: we don't use ``/*` style comments within functions. Instead just start each line with ``//``.